crates/fuel-core/src/state/historical_rocksdb.rs

```diff
@@ -735,11 +1034,245 @@ mod tests {
         assert_eq!(result, Ok(1));
         let entries = historical_rocks_db
             .db
-            .iter_all::<ModificationsHistory<OnChain>>(None)
+            .iter_all::<ModificationsHistoryV2<OnChain>>(None)
             .collect::<Vec<_>>();
         assert_eq!(entries.len(), 0);
     }
 
+    #[test]
```

Comment:
Done: 9f050e5aea60aa5b7d2678a45366715d7a2b949f 
(Although for these tests `Given/When/Then` seem not to work well since we test multiple things pre, mid, and post migration